volume/local/local_unix.go

```diff
@@ -71,6 +70,30 @@ func (v *localVolume) mount() error {
 	if v.opts.MountDevice == "" {
 		return fmt.Errorf("missing device in volume options")
 	}
+	r1, _ := regexp.Compile(`\baddr=*`)
+	r2, _ := regexp.Compile(`\bnfs\b`)
+	if r1.MatchString(v.opts.MountOpts) && r2.MatchString(v.opts.MountType) {
+		if addrValue := getAddress(v.opts.MountOpts); addrValue != "" {
+			ipAddr, err := net.ResolveIPAddr("ip", addrValue)
+			if err == nil {
+				v.opts.MountOpts = strings.Replace(v.opts.MountOpts, addrValue, ipAddr.String(), 1)
+			} else {
+				return errors.Wrapf(err, "Unable to resolve hostname")
+			}
```

Comment:
You can do

```go
if err != nil {
    return errors.Wrapf(err, "unable to resolve hostname"
}
```

So that you don't need an "else".

Perhaps include the actual hostname in the error message, as that may be valuable information